lib/_content.erb

```diff
@@ -0,0 +1,22 @@
+        <section class="doc-preview">
+                 <% if !@site_map['Documents'].nil? %>
+                        <%- @site_map['Documents'].each do |categories| -%>
+                                <%- categories.each do |key, value| -%>
+                                        <%- first_entry = true -%>
+                                        <%- value.each do |key2| -%>
```

Comment:
For the following case where we have the key for the document section inside the yml but no value, we should check if `value` is nil or not. Otherwise, we get an error for the each function

```
Documents:
  - Minio Server:


  - Minio Client:
      - Minio Client Quick Start Guide:
        Link: https://raw.githubusercontent.com/minio/mc/master/README.md
```

Reply:
I know this case is not handled.  We have to handle this case and say we need a value for the first key which is the default landing page also. I will open a bug for this and lets have a discussion on what we want to do when the first key has an empty nil value. This will also impact the start page that shows up as the first page.